com.beavernotes.beavernotes.yaml

```
@@ -0,0 +1,53 @@
+app-id: com.beavernotes.beavernotes
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: beaver-notes
+finish-args:
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+rename-desktop-file: beaver-notes.desktop
+rename-icon: beaver-notes
+modules:
+  - name: beaver-notes
+    buildsystem: simple
+    build-commands:
+      - ar -x Beaver-notes_*.deb
+      - tar -xf data.tar.xz
+      - mkdir -p $FLATPAK_DEST/opt
+      - cp -r opt/Beaver-notes/* $FLATPAK_DEST/opt
+      - mkdir -p $FLATPAK_DEST/share/applications
+      - cp -r usr/share/applications/* $FLATPAK_DEST/share/applications/
+      - mkdir -p $FLATPAK_DEST/share/icons/hicolor
+      - cp -r usr/share/icons/hicolor/* $FLATPAK_DEST/share/icons/hicolor/
+      - desktop-file-edit --set-key=Categories --set-value="X-Productivity;" $FLATPAK_DEST/share/applications/beaver-notes.desktop
+      - desktop-file-edit --set-key=Exec --set-value="beaver-notes %U" $FLATPAK_DEST/share/applications/beaver-notes.desktop
+      - install beavernotes.sh $FLATPAK_DEST/bin/beaver-notes
+      - mkdir -p $FLATPAK_DEST/share/metainfo
+      - install com.beavernotes.beavernotes.metainfo.xml $FLATPAK_DEST/share/metainfo/
```

```suggestion
      - install -Dm0644 com.beavernotes.beavernotes.metainfo.xml -t $FLATPAK_DEST/share/metainfo/
```

This doesn't work again previous one was fine.